### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by taxmeifyoucan**
Following paragraph about specification, added simple diagram by Danny Ryan which helps to explain what Ethereum client features.

### CODE REVIEW

File: src/content/developers/docs/nodes-and-clients/index.md [hunk 1]
```diff
@@ -18,7 +18,10 @@ You should understand the concept of a decentralized network before diving deepe
 
 You can see a real-time view of the Ethereum network by looking at this [map of nodes](https://etherscan.io/nodetracker).
 
-Many [implementations of Ethereum clients](/developers/docs/nodes-and-clients/#clients) exist in a variety of languages. What these client implementations have in common is they all follow a formal specification. This specification dictates how the Ethereum network and blockchain functions.
+Many [implementations of Ethereum clients](/developers/docs/nodes-and-clients/#clients) exist in a variety of languages. What these client implementations have in common is they all follow a formal specification. This specification dictates how the Ethereum network and blockchain functions. 
+
+![Eth1x client](https://storage.googleapis.com/ethereum-hackmd/upload_1b9f852c420c76274b0a899d4e7f39fb.png)
```
Comment (samajammin) [inline src/content/developers/docs/nodes-and-clients/index.md:23]: Please add this image file directly to the project (in this same directory) so we host it ourselves. See other image files within this directory (e.g. archive-sync.png) on how this is done.

**Review by samajammin — CHANGES_REQUESTED** [review]
Thanks for this! One change requested - please let me know if you need assistance.

**Comment by ryancreatescopy** [comment]
@taxmeifyoucan just a bump on this. Would be good to merge this next week. Let me know if you need any help.

File: src/content/developers/docs/nodes-and-clients/index.md (hunk 1 above)
Comment (taxmeifyoucan) [reply src/content/developers/docs/nodes-and-clients/index.md]: Sorry, I wasn't sure about uploading files. Corrected!

**Comment by taxmeifyoucan** [comment]
@ryancreatescopy Updated, hope everything is ok now, ready to merge.

**Review by samajammin — APPROVED** [review]
Thanks for updating!

**Pull request merged**